### PR TITLE
Case study

### PR DESCRIPTION
**Pull request by surbhi90**

### CODE REVIEW

File: sites/all/modules/features/adb_case_study/adb_case_study.module [hunk 1]
```diff
@@ -300,9 +307,9 @@ function adb_case_study_preprocess_node(&$variables) {
           // In Institutions and stakeholder section we have field collection under field
           // collection so we have to relooping aging for inner ettity_id's.
           $sub_section_detail = entity_load('field_collection_item', array((int) $data['value']));
-          $institue_output .= '<div class="detail">';
+          
           foreach ($sub_section_detail as $key => $field_value) {
-            $institue_output .= '<div class="cateogry-detail">';
+            // $institue_output .= '<div class="category-detail">';
```
Comment (subhojit777) [inline sites/all/modules/features/adb_case_study/adb_case_study.module]: Remove this.

File: sites/all/modules/features/adb_case_study/adb_case_study.module [hunk 2]
```diff
@@ -315,15 +322,16 @@ function adb_case_study_preprocess_node(&$variables) {
...
             // If not url is put in field so field content visible simple label field.
             else {
-              $institue_output .= '<div class="detail-title">' .
+              $institue_output .= '<p>' .
                 $field_value->field_value['und'][0]['value'] .
-                '</div>';
+                '</p>';
             }
-            $institue_output .= '</div>';
+            // $institue_output .= '</div>';
```
Comment (subhojit777) [inline sites/all/modules/features/adb_case_study/adb_case_study.module]: Remove this.

File: sites/all/themes/custom/adb/sass/pages/_caseStudy.scss [hunk 3]
```diff
@@ -82,20 +85,71 @@
...
+  }
 }
-.section-heading {
-  @extend %seconday-heading;
+.social-share {
+  li {
+    display: inline-block;
+    margin: 0 5px;
+    &:first-child {
+	  margin-left: 0;
```
Comment (subhojit777) [inline sites/all/themes/custom/adb/sass/pages/_caseStudy.scss]: Correct the indentation.

File: sites/all/themes/custom/adb/sass/pages/_caseStudy.scss [hunk 4]
```diff
@@ -82,20 +85,71 @@
...
+    background-color: $light-grey-background;
+  }
+  p {
+    font-size: ($base-font)+2;
+    line-height: ($base-font)+10;
+    margin-bottom: 30px;
+    color: $black;
+  }
+  img,iframe {
+  	max-width: 100%;
```
Comment (subhojit777) [inline sites/all/themes/custom/adb/sass/pages/_caseStudy.scss]: Correct this indentation

File: sites/all/themes/custom/adb/sass/pages/_caseStudy.scss [hunk 5]
```diff
@@ -156,21 +210,37 @@
 /*
 **Project Snapshot Date Section
 */
-.project-year {
+.project-date {
   display: inline-block;
+  width: 28%;
+  color: $text;
+  &:last-of-type .project-year{
+  	&:after {
```
Comment (subhojit777) [inline sites/all/themes/custom/adb/sass/pages/_caseStudy.scss]: Correct this indentation.

File: sites/all/themes/custom/adb/sass/pages/_caseStudy.scss (hunk 3 above)
Comment (surbhi90) [inline sites/all/themes/custom/adb/sass/pages/_caseStudy.scss]: Corrected

File: sites/all/themes/custom/adb/sass/pages/_caseStudy.scss (hunk 4 above)
Comment (surbhi90) [inline sites/all/themes/custom/adb/sass/pages/_caseStudy.scss]: corrected

**Pull request merged**